### DISCUSSION TITLE
latency of WiFibroadcast

### DISCUSSION THREAD

**Discussion by didadidadi5** [svpcom/wfb-ng#155] [General]
有时我感觉很迷惑，为何传输h264的视频wifibroadcast的延时还不如直接wifi的udp rtp，都是用的相同的wifi网卡。我用wifi 路由器udp rtp 100ms左右延时，用befinitiv 的wifibroadcast延时200ms左右。用svpcom的wifibroadcast 大约400ms延时。
有时我感觉很迷惑，为何传输h264的视频wifibroadcast的延时还不如直接wifi的udp rtp，都是用的相同的wifi网卡。我用wifi 路由器udp rtp 100ms左右延时，用befinitiv 的wifibroadcast延时200ms左右。用svpcom的wifibroadcast 大约400ms延时。
Sometimes I feel very confused. Why is the delay of transmitting H264 video WiFi broadcast not as good as that of direct WiFi UDP RTP? They all use the same WiFi network card. I use the WiFi router UDP RTP with a delay of about 100ms and the WiFi broadcast of befinitiv with a delay of about 200ms. Use the WiFi broadcast of svpcom with a delay of about 400ms.

**Comment by svpcom** [comment]
Normal WiFi use ACK's and retransmissions to avoid packet loss. WFB use FEC. You can tune FEC settings to reduce block size (N value) to decrease delay in case of packet loss. WFB transmission delay is < 30ms. You can test it with ping via WFB tunnel
